region/crateria/west/Terminator Room.json

```diff
@@ -130,8 +130,11 @@
         {"resetRoom": {
           "nodes": [1, 2]
         }},
-        {"refill": ["Energy", "Missile"]},
-        {"partialRefill": {"type": "Super", "limit": 10}}
+        {"cycleFrames": 900}
```

Comment:
15s seems fast with how hard to hit the enemies are.  
Adding wave/spazer/plasma would also help.